R/FilteredData.R

```diff
@@ -552,23 +554,27 @@ FilteredData <- R6::R6Class( # nolint
       tags$div(
         id = id, # not used, can be used to customize CSS behavior
         class = "well",
+        include_js_files(pattern = "togglePanelItem"),
         tags$div(
-          class = "filter-panel-active-header",
-          tags$span("Active Filter Variables", class = "text-primary mb-4"),
-          private$ui_available_filters(ns("available_filters")),
-          actionLink(
-            inputId = ns("minimise_filter_active"),
-            label = NULL,
-            icon = icon("angle-down", lib = "font-awesome"),
-            title = "Minimise panel",
-            class = "remove_all pull-right"
-          ),
-          actionLink(
-            inputId = ns("remove_all_filters"),
-            label = "",
-            icon("circle-xmark", lib = "font-awesome"),
-            title = "Remove active filters",
-            class = "remove_all pull-right"
+          # class = "filter-panel-active-header",
+          style = "display: flex; justify-content: space-between;",
```

Comment:
There are a lot of inline styles added because when we work on the sidebar panel redesign we would have to work on a lot of CSS and inline CSS makes it easier to know what style is added for this element without switching back and forth.
CSS is added only is cases where it was necessary because we needed to switch class in the JS to change the style and inline CSS could not help there,